src/main/java/uk/gov/companieshouse/web/pps/config/PenaltyConfigurationProperties.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.companieshouse.web.pps.config;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+import uk.gov.companieshouse.web.pps.PPSWebApplication;
+import uk.gov.companieshouse.web.pps.util.PenaltyReference;
+
+@Configuration
+@ConfigurationProperties("penalty")
+public class PenaltyConfigurationProperties {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(PPSWebApplication.APPLICATION_NAME_SPACE);
+
+    private List<String> allowedRefStartsWith = new ArrayList<>();
+    private String refStartsWithPath;
+    private String enterDetailsPath;
+    private String bankTransferWhichPenaltyPath;
+    private String bankTransferLateFilingDetailsPath;
+    private String bankTransferSanctionsPath;
+
+    public List<String> getAllowedRefStartsWith() {
+        return allowedRefStartsWith;
+    }
+
+    public void setAllowedRefStartsWith(List<String> allowedRefStartsWith) {
+        this.allowedRefStartsWith = allowedRefStartsWith
+                .stream()
+                .map(startsWith -> {
+                    try {
```

Comment:
Is it worth chunking this up in a separate method so it's not nested in the stream?

```suggestion
                    return methodName();
                    
      ...
      
      private String methodName { try/catch...
      
```

Reply:
Done in commit [38b7726](https://github.com/companieshouse/penalty-payment-web/pull/160/commits/38b7726568166183218151ed3c0e8c16cd4f4f84)

Reply:
Ah, I meant more remove just the try/catch from inside stream().map() into its own method. All good though, It's not a biggie and your change also helps with readability anyways.